argo-cd-apps/base/all-clusters/infra-deployments/horreum/horreum.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
This should be placed under argo-cd-apps/base/member/infra-deployments (you can copy/paste the kustomize file from another directory in that path and just change the file names). 

Right now it will deploy also to the host cluster, but we want to deploy only to the members.

Reply:
Thank you! Should be resolved with https://github.com/redhat-appstudio/infra-deployments/pull/2798/commits/686faa6dc2e242148390bb7a825878e8e43b1b72